### ISSUE TITLE
[Feature request] Home Assistant Add-On

### ISSUE THREAD

**Issue by agisofttm** [toxuin/mdns-tunneller#11]
Hi @toxuin, 

Is there a chance your container to became Home Assistant AddOn as this documentation? 
https://developers.home-assistant.io/docs/add-ons/tutorial/

We can use your GitHub as repository, so there is no need to register add-on as official.

Thanks

**Comment by toxuin** [comment]
Hello!

It sounds like a great idea!

I don’t use Home Assistant in a managed configuration (= with addons), but I would definitely welcome such improvement as a contribution.

I can help with implementation to the best of my ability.

@agisofttm is that something you’d be interested in trying to implement?

**Comment by agisofttm** [comment]
@toxuin Yes Dude, did you know - your mDNS-tunneler working flawlessly on all of my offices, where I have Linux OS with computer, but I've a huge problem where I don't have those machines and have only Home Assistant which is also Linux based OS, but is locked inside Docker Containers. If You manage to do this that will be great, because a huge amount of Peapoles use HA for their home automation, and they have only this actually :-)))))))) I can help with everything if you can do it, and of you need some access to that system or I can test it as well :-)
